packages/office-ui-fabric-react/src/common/_themeOverrides.scss

```diff
@@ -24,7 +24,6 @@ $ms-color-neutralLighterAlt: "[theme:neutralLighterAlt, default: #f8f8f8]";
 $ms-color-accent: "[theme:accent, default: #0078d4]";
 $ms-color-white: "[theme:white, default: #ffffff]";
 $ms-color-whiteTranslucent40: "[theme:whiteTranslucent40, default: rgba(255,255,255,.4)]";
-$ms-color-yellowDark: "[theme:yellowDark, default: #d29200]";
```

Comment:
It looks like `yellowDark` was added in this PR https://github.com/OfficeDev/office-ui-fabric-react/pull/7976 for fluent folder styling. Should we add it to IPalette?